MapboxNavigation/Constants.swift

```diff
@@ -15,6 +15,9 @@ public let RouteControllerProgressDidChange = Notification.Name(MBRouteControlle
 public let RouteControllerAlertLevelDidChange = Notification.Name(MBRouteControllerAlertLevelDidChange)
 public let RouteControllerShouldReroute = Notification.Name(MBRouteControllerShouldReroute)
 
+public let RouteControllerDidReroute = Notification.Name(MBRouteControllerDidRerouteKey)
+public let RouteControllerDidRerouteNotifcationRouteProgressKey = Notification.Name(MBRouteControllerDidRerouteNotificationRouteProgressKey)
```

Comment:
Maybe we should rename `MBRouteControllerAlertLevelDidChangeNotificationRouteProgressKey` to be more generic and reuse the consolidated key in the new notification.